### DISCUSSION TITLE
We need iOS port...

### DISCUSSION THREAD

**Discussion by Fanatic4** [skyline-emu/skyline#1988] [Ideas]
I mean I know how hard it is to run this emulator on iOS but please we need to try something. There is huge audience for people who emulate on iOS. I mean retroarch did then why can’t this emulator. I strongly encourage support for iOS as it is way more powerful and fluid than android. I hope this ignited some curiosity towards the devs and one other thing, English is not my first language so expect some grammatical errors. Keep up the good work!

**Comment by AndresNorm** [comment]
Huh?
Why not buy an Android then?, instead of demanding for a port.
Or why not just stick to your Switch Console

**Reply by Fanatic4** [reply]
What if I didn’t have a switch console or what if I don’t have an Android then what ? Oh ok lemme just spend couple hundred bucks for an sd 8 first gen phone because there is no ns emulator on My current iPhone 12 pro phone that I bought. Yea didn’t think so

**Reply by nekomekoraiyuu** [reply]
@Fanatic4  Lmao then you're admitting to piracy if;
you do not have a switch and want to emulate
also you cannot use the application without owning a switch legally

**Reply by JERD001** [reply]
Would say otherwise tbh, emulation on iOS doesn't have much support. iOS doesn't have much support over ~2000 era consoles onwards, there's not much interest nor fanbase to do a project like that. Even if there were iOS main problem still their limitations like JIT isn't available unless you jailbreak (a niche under a niche), appstore limiting the availability of emulators, use of non standard graphics API, etc. If someone wants to make a port the code is there and it'll be a fork like DolphiniOS but never official.

**Reply by Fanatic4** [reply]
It doesn’t need to be “official” only workable. Other than that thank you for that info.

**Comment by PixelyIon** [comment]
This question has already been answered in #387; this is extremely hard to do due to the limitations that Apple imposes on iOS.

**Comment by Mortuarium11** [comment]
Real gamers use 
Pc 
Android 
And PlayStation
